products/terraform/sidebars.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+  terraformpanos: [
+    "terraform/docs/panos/intro",
+    {
+      type: "category",
+      collapsed: true,
+      label: "Getting Started Tutorials",
+      items: [
+        "terraform/docs/panos/tutorials/setup",
+        {
+          type: "category",
+          collapsed: true,
+          label: "Information Gathering Tasks",
+          items: ["terraform/docs/panos/tutorials/info-gather-sysinfo"],
+        },
+      ],
+    },
+    {
+      type: "category",
+      collapsed: true,
+      label: "How-To Guides",
+      items: [
+        "terraform/docs/panos/guides/commits",
+        "terraform/docs/panos/guides/auditcomments",
+      ],
+    },
+    {
+      type: "category",
+      collapsed: true,
+      label: "Background Information",
+      items: [
+        "terraform/docs/panos/info/background",
+        "terraform/docs/panos/info/connectivity",
+        "terraform/docs/panos/info/parallelism",
+        "terraform/docs/panos/info/create-before-destroy",
+        "terraform/docs/panos/info/panos-versions",
+      ],
+    },
+    {
+      type: "category",
+      collapsed: true,
+      label: "Reference",
+      items: ["terraform/docs/panos/reference/provider"],
+    },
+  ],
+  terraformcloudngfw: [
+    "terraform/docs/cloudngfw/intro",
+    {
+      type: "category",
+      collapsed: false,
+      label: "AWS",
+      items: [
+        // {
+        //   type: "category",
+        //   collapsed: true,
+        //   label: "Getting Started Tutorials",
+        //   items: ["terraform/cloudngfw/aws/tutorials/tutorial"],
+        // },
+        // {
+        //   type: "category",
+        //   collapsed: true,
+        //   label: "How-To Guides",
+        //   items: ["terraform/cloudngfw/aws/guides/guide"],
+        // },
+        // {
+        //   type: "category",
+        //   collapsed: true,
+        //   label: "Background Information",
+        //   items: ["terraform/cloudngfw/aws/info/info"],
+        // },
+        {
+          type: "category",
+          collapsed: true,
+          label: "Reference",
+          items: ["terraform/docs/cloudngfw/aws/reference/provider"],
+        },
+      ],
+    },
+  ],
+};
```

Comment:
I can see the two ```sidebars.js``` files for the two Terraform providers have been consolidated. It still works though! How does it work, I can't see how each section (```terraformpanos``` and ```terraformcloudngfw```) are referenced? It's important for me to understand because the ```terraformcloudngfw```label the secions section in ```module.exports``` might want to change so I want to know if that will break anything :-)

Reply:
Hi James, `sidebars.js` can contain more than one sidebar and the association automatically happens for each doc you reference in a sidebar. Those "labels" you are referring to are only referenced in the backend, when the association is made. Typically, you can change them without breaking anything so you should be ok.

More reading: https://docusaurus.io/docs/sidebar/multiple-sidebars